libraries/wasm/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "iota_identity_wasm"
+version = "0.0.1"
+authors = ["Team Identity"]
+edition = "2018"
+repository = "https://github.com/iotaledger/identity.rs"
+homepage = "https://www.iota.org"
+license = "MIT"
+
+[lib]
+crate-type = ["cdylib"]
+
+# TODO: Discover this:
```

Comment:
??

Reply:
Probably the wasm-opt thing, I tried it with other values, but it looks like it's not supported atm and it's currently recommended just to set it to false